src/apps/common/submodules/navigationWizard/navigationWizard.js

```diff
@@ -30,44 +61,60 @@ define(function(require) {
 						doneButton: _.get(args, 'doneButton', '')
 					},
 					submodule: 'navigationWizard'
-				}));
+				})),
+				navigationWizardFlags = {
+					cancelConfirmation: false,
+					currentStep: 0,
+					validateOnStepChange: false,
+					wizardArgs: _.merge(args, {
```

Comment:
This is because `args` has a reference to the application that invoked the wizard, through the `thisArg` property. If I merge it to an empty object, the whole app object will be cloned, so we will hae 2 instances of it.

Reply:
On a second thought, I think I'll refactor this to exclude the `thisArg` from the `merge`.